buildrunner/__init__.py

```diff
@@ -124,20 +118,19 @@ def _get_config_context(self, ctx=None, global_env=None):
 
     def __init__(
         self,
-        build_dir,
-        build_results_dir=None,
+        *,
```

Comment:
Did you mean to have this `*` here?

Reply:
Yes, it makes all of the following parameters required as keyword args (which matches how this is actually used in the code). I should probably actually remove the default values for all of these at the same time since they are meaningless now (which they already were, hence the `*`).